src/transforms/wasm.rs

```diff
@@ -137,157 +150,148 @@ mod tests {
         Ok(Some(event))
     }
 
-    #[test]
-    fn protobuf_happy() -> crate::Result<()> {
+    async fn test_config(config: &str, input: &str, output: &str) {
+        let transform = parse_config(config).expect("could not init transform");
+
+        let input = vec![parse_event_artifact(input)
+            .expect("could not load input protobuf")
```

Comment:
This error is about protobufs but it's used in the add fields test which does not use protobufs. I am confused.

Reply:
Doh, yeah, I'll update.